pkg/features/kube_features.go

```diff
@@ -636,6 +636,7 @@ const (
 
 	// owner: @derekwaynecarr
 	// alpha: v1.20
+	// beta: v1.21
```

Comment:
we need to update this to 1.22, if you want i can just open a separate pr.

Reply:
Done @derekwaynecarr